### PR TITLE
US2: Create/POST

### PR DESCRIPTION
**Pull request by cananozyol**

### CODE REVIEW

File: backend/src/test/java/de/neuefische/readandmeet/backend/BackendApplicationTest.java [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
+
+class BackendApplicationTest {
+
     @Test
     void contextLoads() {
         Assertions.assertTrue(true);
     }
-}
+
+}
```
Comment (Pia0912) [inline backend/src/test/java/de/neuefische/readandmeet/backend/BackendApplicationTest.java:14]: please add an empty line if you did not already do so 😆

File: backend/src/test/java/de/neuefische/readandmeet/backend/controller/MeetingControllerTest.java [hunk 2]
```diff
@@ -47,4 +48,29 @@ void expectMeetingList_whenGETMeetingList() throws Exception {
...
+
+        //THEN
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].id").isNotEmpty())
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].title").value("book"))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].date").value("2023-08-08"))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].location").value("online"));
+    }
 }
```
Comment (Pia0912) [inline backend/src/test/java/de/neuefische/readandmeet/backend/controller/MeetingControllerTest.java:76]: a blank line seems to be missing here as well 👀

File: backend/src/test/java/de/neuefische/readandmeet/backend/service/MeetingServiceTest.java [hunk 3]
```diff
@@ -31,4 +32,35 @@ void expectListOfAllMeetings() {
...
+        when(uuIdService.getRandomId()).thenReturn("123");
+        when(meetingRepo.insert(expected)).thenReturn(expected);
+        Meeting actual = meetingService.add(meetingWithoutId);
+
+        //THEN
+        assertEquals(expected, actual);
+        verify(uuIdService).getRandomId();
+        verify(meetingRepo).insert(expected);
+    }
 }
```
Comment (Pia0912) [inline backend/src/test/java/de/neuefische/readandmeet/backend/service/MeetingServiceTest.java:66]: add an empty line 👀

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -1,12 +1,39 @@
...
 }
+
+const StyledApp = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 1.1em;
+  padding-top: 2em;
+`;
```
Comment (Pia0912) [inline frontend/src/App.tsx:39]: add an empty line 👀

File: frontend/src/hooks/useFetch.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    postMeeting: (requestBody: MeetingWithoutId) => {
+        const { fetchMeetings } = get();
+        axios
+            .post("/api/meetings", requestBody)
+            .then(fetchMeetings)
+            .catch(console.error);
+    },
+
+}));
```
Comment (Pia0912) [inline frontend/src/hooks/useFetch.ts:33]: add an empty line 👀

File: frontend/src/pages/AddPage.tsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    );
+}
+
+const StyledAdd = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 1.1em;
+  padding-top: 2em;
+`;
```
Comment (Pia0912) [inline frontend/src/pages/AddPage.tsx:81]: add an empty line 👀

**Review by Pia0912 — APPROVED** [review]
Looks really nice Canan! Also really nice done with the custom hooks  🚀👍🏻

File: backend/src/test/java/de/neuefische/readandmeet/backend/controller/MeetingControllerTest.java (hunk 2 above)
Comment (cananozyol) [reply backend/src/test/java/de/neuefische/readandmeet/backend/controller/MeetingControllerTest.java:76]: hups 🙃.. thank you adlerauge 🦅

**Pull request merged**